### PR TITLE
Fix web selections

### PR DESCRIPTION
**Pull request by andybalaam**

### CODE REVIEW

File: platforms/web/lib/useWysiwyg.test.tsx [hunk 1]
```diff
@@ -31,10 +31,57 @@ const Editor = forwardRef<HTMLDivElement>(function Editor(props, forwardRef) {
     contentEditable={isWysiwygReady} />;
 });
 
+function toContainHtml(
+    // TODO: remove lint ignores? I can't find the right import to be able to use the types.
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    this: any, /* MatcherState */
```
Comment (florianduros) [inline platforms/web/lib/useWysiwyg.test.tsx:37]: I'm afraid, that we need to add the jest types and use `jest.MatcherContext` here

**Review by florianduros — APPROVED** [review]
Look goods ! 

Some comments about the type

File: platforms/web/lib/useWysiwyg.test.tsx [hunk 2]
```diff
@@ -31,10 +31,57 @@ const Editor = forwardRef<HTMLDivElement>(function Editor(props, forwardRef) {
...
+    editor: HTMLDivElement,
+    html: string,
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+): any /* ExpectationResult */ {
```
[the remark this answers is not in the document]
Comment (andybalaam) [reply platforms/web/lib/useWysiwyg.test.tsx:41]: Yes, done!

File: platforms/web/lib/useWysiwyg.test.tsx (hunk 1 above)
Comment (andybalaam) [reply platforms/web/lib/useWysiwyg.test.tsx:37]: Thanks, how would I do that?

Comment (florianduros) [reply platforms/web/lib/useWysiwyg.test.tsx:37]: // Adding jest type dev dependencies
- `yarn add -D @types/jest`
// Add jest in the global types
- Add `jest`in the `types`array in the tsconfig
// Use the MatcherContext type for the `this` type
- `this: jest.MatcherContext`

Comment (andybalaam) [reply platforms/web/lib/useWysiwyg.test.tsx:37]: Thank you!

Comment (andybalaam) [reply platforms/web/lib/useWysiwyg.test.tsx]: Done!

**Pull request merged**